src/commands/create.js

```diff
@@ -52,9 +56,58 @@ function create (opts) {
       if (err && err.name === 'ExistsError') return bus.emit('exit:warn', 'Archive already exists.')
       if (err) return bus.emit('exit:error', err)
 
-      // TODO: dat.json creation/write key
-      state.dat = dat
-      bus.emit('dat')
+      // create before import
+      var datjson = DatJson(dat.archive, { file: path.join(opts.dir, 'dat.json') })
+      fs.readFile(path.join(opts.dir, 'dat.json'), 'utf-8', function (err, data) {
+        if (err) return doPrompt()
+        data = JSON.parse(data)
+        debug('read existing dat.json data', data)
+        doPrompt(data)
+      })
+
+      function doPrompt (data) {
+        if (!data) data = {}
+
+        var schema = {
+          properties: {
+            title: {
+              description: chalk.magenta('Dat Title'),
+              default: data.title || '',
+              // pattern: /^[a-zA-Z\s\-]+$/,
+              // message: 'Name must be only letters, spaces, or dashes',
+              required: false
+            },
+            description: {
+              description: chalk.magenta('Dat Description'),
+              default: data.description || ''
+            },
+            doImport: {
+              description: chalk.magenta('Would you like to import your files?'),
+              default: 'yes'
+            }
+          }
+        }
+        prompt.message = chalk.green('> ')
+        prompt.delimiter = '' // chalk.cyan('')
+        prompt.start()
+        prompt.get(schema, writeDatJson)
+
+        function writeDatJson (err, results) {
+          if (err) return console.log(err.message) // prompt error
```

Comment:
Yes! At first I thought this may be prompt errors, e.g. if you get something that doesn't match the required pattern, but I think that is handled differently.

> would the bus.emit('exit:error', err) do that?

yep